napoli-pain/src/components/order_details/order_details.rs

```
@@ -90,6 +105,14 @@ impl Component for OrderDetails {
                 self.order = Some(order);
                 true
             }
+            Self::Message::StreamingConnected(stream) => {
+                ctx.link()
+                    .send_stream(stream.map(|single_order_reply_result| {
+                        let o: npb::Order = single_order_reply_result.ok().unwrap().order.unwrap();
```

`.ok().unwrap()` discards the error message and just panics with `None` which is less descriptive.  `.unwrap()` the result directly instead.